dev/ansible/02_put_variables_consul.yml

```diff
@@ -0,0 +1,12 @@
+- name: Put repo value in Consul Key Value Store
```

Comment:
could you include this line on your PR title , so It will make  more understandable "The  Purpose of PR"

Reply:
The main purpose with this PR is Secret Management, but I can add a sentence about Consul Key Value Store 👍